templates/ra_rdcb_fileserver_standalone.template.cfn.yaml

```
@@ -189,6 +193,10 @@ Parameters:
     AllowedPattern: '^$|.*[@].*'
     Description: (Optional) Email address to subscribe to notifications and alarms
     Type: String
+  SnapShotFrequency:
```

Matching the amazon docs for case on `Snapshot`...
```suggestion
  SnapshotFrequency:
```